javascript/javascript3/week3/secondExercise.js

```diff
@@ -0,0 +1,11 @@
+
+function myPromise(resolveAfter) {
+  return new Promise((resolve, reject) => {

```

Comment:
seems like there's some week2 here in this PR. I'm just gonna ignore those files